CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 ## Unreleased
 
-### Build
-* Build with go 1.23.7
+
+## 0.22.1 (May 30, 2025)
+
+## Build
```

Comment:
Small nit: But let's make sure that `### Build` is the same header size as `### Dependency updates`, as opposed to the version (`##`)